File: nagasena/src/org/openexi/proc/grammars/EventTypeSchema.java
package org.openexi.proc.grammars;

import org.openexi.proc.common.EventType;
import org.openexi.proc.common.EventTypeList;

public final class EventTypeSchema extends EventType {
  
  public final int nd;

  EventTypeSchema(int nd, String uri, String name, int uriId, int nameId, byte depth, 
      EventTypeList eventTypeList, byte itemType, EXIGrammar subsequentGrammar) {
    super(uri, name, uriId, nameId, depth, eventTypeList, itemType, NOT_AN_EVENT, subsequentGrammar);
    this.nd = nd;
  }

}

File: nagasenacs/src/Nagasena/Proc/Grammars/EventTypeSchema.cs
using EventDescription_Fields = Nagasena.Proc.Common.EventDescription_Fields;
using EventType = Nagasena.Proc.Common.EventType;
using EventTypeList = Nagasena.Proc.Common.EventTypeList;

namespace Nagasena.Proc.Grammars {

  /// <exclude/>
  public sealed class EventTypeSchema : EventType {

    public readonly int nd;

    internal EventTypeSchema(int nd, string uri, string name, int uriId, int nameId, sbyte depth, EventTypeList eventTypeList, sbyte itemType, EXIGrammar subsequentGrammar) :
      base(uri, name, uriId, nameId, depth, eventTypeList, itemType, EventDescription_Fields.NOT_AN_EVENT, subsequentGrammar) {
      this.nd = nd;
    }

  }

}